pkg/scripts/install.sh

```diff
@@ -53,14 +53,14 @@ EOF
 process_arguments() {
   # set the default value for config file if is not provided
   if [ -z "${CONFIG_FILE}" ]; then
-    CONFIG_FILE="${SCRIPT_PATH}/../../config.tfvars"
+    CONFIG_FILE="config.tfvars"
   else
     if [[ ! -f "${CONFIG_FILE}" ]]; then
       echo "Terraform configuration file '${CONFIG_FILE}' is not found!"
       show_help
     fi
-    OVERRIDE_CONFIG_FILE="-var-file=${CONFIG_FILE}"
   fi
+  OVERRIDE_CONFIG_FILE="-var-file=${CONFIG_FILE}"
```

Comment:
No need to move this out of if block. If there is no other config file is passed then we don't have to set `OVERRIDE_CONFIG_FILE` and is good to leave this blank. In this case it uses default config file which is `config.tfvars` from root folder of repo.

Reply:
It won't as Carlos experienced firsthand. Because we have removed `.auto` from the config file name, the  config won't be picked up.

Reply:
It is not true, I just test it again and the default `config.tfvars` is picked if the replacement config file is not provided.

Reply:
Awesome, if this works now after the changes, I can close this PR.